pandas/tests/frame/test_alter_axes.py

```diff
@@ -193,6 +193,11 @@ def test_set_index_pass_arrays_duplicate(self, frame_of_index_cols, drop,
         df.index.name = index_name
 
         keys = [box1(df['A']), box2(df['A'])]
+
```

Comment:
make this a separate test

Reply:
Why? It's nicely parametrized, and breaking that out would be a huge hassle. All I'm doing is xfailing as appropriate.

Reply:
because its very messy otherwise. i guess if its too difficult, ok, but add a TODO